Microsoft.Toolkit.Uwp.SampleApp/Styles/Generic.xaml

```diff
@@ -0,0 +1,997 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.SampleApp.Controls"
+    xmlns:toolkitControls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Animations.Behaviors"
+    xmlns:animations="using:Microsoft.Toolkit.Uwp.UI.Animations"
+    xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="ms-appx:///Controls/ExtendedHamburgerMenu.xaml" />
+        <ResourceDictionary Source="ms-appx:///Controls/CodeRenderer.xaml" />
+        <ResourceDictionary Source="ms-appx:///Styles/Custom/PivotHeaderItemUnderlineStyle.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <!--  Styles  -->
+    <Style x:Key="AboutHyperlinkButtonStyle"
```

Comment:
Could be handy to pull some of these larger ones out into the Styles/Custom directory as well, no?